_extras/glossary.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Glossary
+---
+
+## ESMValTool Glossary
+
+Here, there are some definitions used throughout this tutorial:
+
+- **Amon**: Monthly atmospheric data
+
+- **CMIP**: The Coupled Model Intercomparison Project (CMIP) aims at better understanding past, present and future climate changes arising from natural, unforced variability or in response to changes in radiative forcing in a multi-model context. For more information, check the [WCR-Climate webpage](https://www.wcrp-climate.org/).
+
+- **CMOR**: CMOR stands for [*Climate Model Output Rewriter* library](https://pcmdi.github.io/cmor-site/index.html). It comprises a set of C-based functions, with bindings to both Python and FORTRAN 90, that can be used to produce CF-compliant NetCDF files that fulfill the requirements of many of the climate community's standard model experiments.
+
+- **dataset**: In a recipe, dataset refers to the name of the model or observation data, e.g. HadGEM2-ES
+
+- **diagnostic**: A diagnostic script is the last step in a recipe to convert the pre-processed input data to the desired output like plots or NetCDF files.
```

Comment:
```suggestion
- **diagnostic**: In a recipe, the diagnostic section(s) define the tasks that will be executed when running the recipe. It can also include a diagnostic script that converts the preprocessed input data to the desired output like plots or NetCDF files.
```